binary_search_tree/binary_search_tree.py

```diff
@@ -37,7 +37,14 @@ def rec(value, current_node):
         return rec(target, self)
 
     def get_max(self):
-        pass
+        maxValue = -float("inf")
```

Comment:
Nice job incorporating what we learned in the buying_stocks problem we had in algo's